internal/host/validator.go

```diff
@@ -583,7 +583,7 @@ func (v *validator) areImagesAvailable(c *validationContext) ValidationStatus {
 
 	for _, imageStatus := range imageStatuses {
 		if imageStatus.Result == models.ContainerImageAvailabilityResultFailure ||
-			imageStatus.DownloadRate < ImageStatusDownloadRateThreshold {
+			(imageStatus.SizeBytes > 0 && imageStatus.DownloadRate < ImageStatusDownloadRateThreshold) {
```

Comment:
Don't we need to check the result?
Also why do we check the sizeBytes?

Reply:
The result is being checked.
In case it's a failure - return false.
Otherwise (=Success) - Check for the download rate.

Only in case there's is a positive image size (which means that measurements were collected) - check the download rate is in the threshold.

Reply:
Sorry, missed that.